src/index.js

```diff
@@ -47,7 +47,17 @@ if (process.env.SENTRY_DSN) {
     DecafAppModule.requires.push('ngRaven');
 }
 
-DecafAppModule.config((appNameProvider, appAuthProvider, potionProvider, decafAPIProvider, modelWSProvider, modelAPIProvider) => {
+function noAuth(appNavigationProvider, moduleName) {
+    let el = appNavigationProvider.navigation.filter((el) => {return el.title === moduleName;})[0];
+    el.authRequired = false;
+}
+
+
+DecafAppModule.config((appNameProvider, appAuthProvider, potionProvider, decafAPIProvider, modelWSProvider, modelAPIProvider, appNavigationProvider) => {
+    noAuth(appNavigationProvider, 'Experiments');
+    noAuth(appNavigationProvider, 'Pools');
+    noAuth(appNavigationProvider, 'Media');
```

Comment:
I'm not a big fan of this function, since it's working with a side effect. I think you could do

    const noAuthModules = ['Experiments', 'Pools', 'Media'];
    appNavigationProvider.navigation
      .filter((app) => noAuthModules.includes)
      .forEach((el) => {
       el.authRequired = false;
     });

It's not too lengthy but it's side effect-free (if you can ever say that about Angular). You can always create a separate config block if you think it's too long.